### ISSUE TITLE
Better URL validation

### ISSUE THREAD

**Issue by NathanBnm** [cassidyjames/ephemeral#93]
In `UrlEntry.vala` at line 45 you say
>`// TODO: Better URL validation`
I think you should use RegEx

**Comment by cassidyjames** [comment]
Yes, that was the original design. I simplified it and added that TODO. :smile: Thanks for documenting it here!
